features/step_definitions/generic_steps.rb

```diff
@@ -18,12 +18,12 @@
 
 Then("the following sections should exist:") do |table|
   table.hashes.each do |row|
-    expect(page).to have_selector(row[:tag], text: /\A#{Regexp.quote(row[:section])}\z/), "expected to find tag \"#{row[:tag]}\" with text: \"#{row[:section]}\""
+    expect(page).to have_selector(row[:tag], text: row[:section]), "expected to find tag \"#{row[:tag]}\" with text: \"#{row[:section]}\""
```

Comment:
I removed the `Regexp.quote` because it was confusing, and it didn't match on the income summary page.

Just matching on the provided string seems to work nicely for all the integration tests.

Reply:
Ok, that makes sense.

I'm happy to leave it if we can figure out why it wouldn't match `Benefits` on that page 🤔

Reply:
@cpjmcquillan am happy to approve this PR if we can just revert this change that removes the regex. Can you put it in a separate branch to discuss so it does not block this PR which we need to get in ASAP. I am just concerned that changing this generic step used in many other features could introduce false positives in future if we remove.

Reply:
Sure 👍